.github/workflows/generate-preview.yml

```diff
@@ -29,6 +29,30 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
 
+      - name: Remove create-preview label
+        uses: actions/github-script@0.9.0
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.issues.removeLabels({
+              issue_number: github.event.pull_request.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              name: ['create-preview']
```

Comment:
Might have messed this up in the original version, I'm not sure this is an array or just a single string